### PR TITLE
OSDOCS-10029# Remove AliCloud CSI Driver

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.16

Issue: https://issues.redhat.com/browse/OSDOCS-10029

Link to docs preview:
https://73734--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi#csi-drivers-supported_persistent-storage-csi
https://73734--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/understanding-persistent-storage#types-of-persistent-volumes_understanding-persistent-storage
https://73734--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/understanding-persistent-storage#pv-access-modes_understanding-persistent-storage
https://73734--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/understanding-persistent-storage#block-volume-support_understanding-persistent-storage

QE review:
- [x] QE has approved this change.

Additional information:

**PTAL**: @dobsonj @duanwei33 @gcharot

### CODE REVIEW

File: modules/persistent-storage-csi-drivers-supported.adoc [hunk 1]
```diff
@@ -29,7 +29,6 @@ and which CSI features they support, such as volume snapshots and resize.
 |===
 |CSI driver |CSI volume snapshots  |CSI cloning  |CSI resize |Inline ephemeral volumes
 ifndef::openshift-dedicated,openshift-rosa[]
```
Comment (mramendi) [inline modules/persistent-storage-csi-drivers-supported.adoc:31]: Looks like you have an empty ifndef/endif block now, maybe remove these lines?

File: _topic_maps/_topic_map.yml [hunk 2]
```diff
@@ -1663,8 +1663,6 @@ Topics:
     File: persistent-storage-csi-migration
   - Name: Detach CSI volumes after non-graceful node shutdown
     File: persistent-storage-csi-vol-detach-non-graceful-shutdown
-  - Name: AliCloud Disk CSI Driver Operator
```
Comment (mramendi) [inline _topic_maps/_topic_map.yml:1666]: Maybe you want to delete the module file? Or is it still included in some other build? 

Also, are you sure there were no xrefs/links to this module apart from the one you remove? Did you do a full text search for IDs in it?

Just some additional checks I would suggest, to be sure everything is in order. If already done, ignore.

**Review by mramendi — COMMENTED** [review]
A couple of possible technical suggestions, obverwie LGTM for peer review. Also suggest adding the following link to the doc links in the description: 

https://73734--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-sc-manage

As I am in peer review induction, @JoeAldinger will finish this review

**Review by mramendi — APPROVED** [review]

File: _topic_maps/_topic_map.yml (hunk 2 above)
Comment (JoeAldinger) [reply _topic_maps/_topic_map.yml:1666]: looks like it shows up here: persistent-storage-csi-sc-manage.adoc

**Review by JoeAldinger — COMMENTED** [review]
a few questions for consideration

**Comment by duanwei33** [comment]
@ropatil010
Could you help take a look?

File: _topic_maps/_topic_map.yml (hunk 2 above)
Comment (JoeAldinger) [reply _topic_maps/_topic_map.yml:1662]: LGTM

**Comment by dobsonj** [comment]
@lpettyjo I think this should be removed as well:
https://github.com/openshift/openshift-docs/blob/main/storage/container_storage_interface/persistent-storage-csi-alicloud-disk.adoc
`git rm storage/container_storage_interface/persistent-storage-csi-alicloud-disk.adoc` ?
That's the only remaining reference I could find.

**Comment by lpettyjo** [comment]
> @lpettyjo I think this should be removed as well: https://github.com/openshift/openshift-docs/blob/main/storage/container_storage_interface/persistent-storage-csi-alicloud-disk.adoc `git rm storage/container_storage_interface/persistent-storage-csi-alicloud-disk.adoc` ? That's the only remaining reference I could find.

@dobsonj I've removed it.

**Comment by dobsonj** [comment]
LGTM, thanks!

**Comment by ropatil010** [comment]
LGTM..!

**Pull request merged**